.github/workflows/run_ci_tests.yml

```diff
@@ -0,0 +1,10 @@
+name: CI
```

Comment:
May CI name can be more explicit to the test done here like prefixing INDEX_GXA_CI or anything you feel more self explanatory.